src/main/java/org/puyallupfamilyhistorycenter/service/websocket/FamilyHistoryCenterSocket.java

```diff
@@ -171,6 +171,13 @@ public UserContext(String userId, String userName, String userEmail, String hash
         }
     }
     
+    private static class Message {
+        public final String cmd;
+        public Message(String cmd) {
+            this.cmd = cmd;
```

Comment:
This looks good for now. In the future we'll need to be able to attach parameters, but that can be in a future change.

Reply:
BTW, the build is failing due to unit tests (I think). I haven't gotten around to fixing them, so that's another task we should add to the backlog.